test/router-options.test.js

```diff
@@ -217,7 +217,7 @@ test('Should supply Fastify request to the logger in frameworkErrors wrapper - F
     }
   )
 })
-
+//
```

Comment:
```suggestion
```

Reply:
in test/router-options.test.js I was temporarily removing tests to determine where the errors were.  So the // was used as a placeholder.

Reply:
If the PR is a work in progress, please mark it as a draft. Otherwise, please remove these extraneous lines.

Reply:
Ok sounds good thanks.